### ISSUE TITLE
Error: read ECONNRESET

### ISSUE THREAD

**Issue by glorya** [bigeyex/brackets-sftp-upload#79]
Still getting this error in the last two days.
I've tried to uninstall and reinstall the plugin, but the error is still there.

**Comment by RobWellington** [comment]
I am having the same issue. I have tried every possible combo of server settings. yours was a year ago...did you work it out.

**Comment by glorya** [comment]
@RobWellington no. Now I have a different error: "Max connections attempt reached". :(

**Comment by Min11Benja** [comment]
I just installed this, and have tried multiple server accounts and combinations, Im using GoDaddy Hosting I thought id be cool to have an ssh via Brackets, but this has taken up way too much time to setup. Its 2017 and I guess this is not solved?
